python/tvm/relay/frontend/mxnet.py

```diff
@@ -384,6 +384,20 @@ def _mx_roi_align(inputs, attrs):
     new_attrs["layout"] = "NCHW"
     return _op.vision.roi_align(inputs[0], inputs[1], **new_attrs)
 
+def _mx_upsampling(inputs, attrs):
+    scale_height = attrs.get_float("scale_height", None)
+    scale_width = attrs.get_float("scale_width", None)
+    if scale_height is None:
+        height = attrs.get_int("height", 1)
+        scale_height = float(height) // inputs[0].shape[2]
+    if scale_width is None:
+        width = attrs.get_int("width", 1)
+        scale_width = float(width) // inputs[0].shape[3]
+    assert scale_height == scale_width
+    scale = scale_width
+    layout = 'NCHW'
+    method = 'BILINEAR'
+    return _op.nn.upsampling(inputs[0], scale=scale, layout=layout, method=method)
```

Comment:
advice to use _op.nn.resize instead which directly takes target height and width and scale factor need not be same too.

Reply:
I didn't see resize under `_op.nn`, there only a `resize` under image which I don't think should be mapped to `image.imresize` from mxnet instead.

Reply:
yes, its _op.image.resize.
Technically upsampling and resize share same topi implementations at the end.

My intention was to avoid ```assert scale_height == scale_width```  even though we could support asymmetric scaling.

Reply:
That makes sense. I'll change accordingly.